sympy/ntheory/generate.py

```diff
@@ -667,6 +661,12 @@ def primerange(a, b):
         Sieve.primerange : return range from already computed primes
                            or extend the sieve to contain the requested
                            range.
+
+        References
+        ==========
+
+        1. https://en.wikipedia.org/wiki/Prime_number
+        2. http://primes.utm.edu/notes/gaps.html
```

Comment:
This should be `.. [1]` and `.. [2]`.

Reply:
Thanks! Missed that!

Btw, is there any consensus if the references always should be numbered or if they should only be numbered if they are explicitly references in the text? I have seen both options.

Reply:
It is better to have numbered references. That way, there is flexibility to cite them with ease in the notes or description section.

Quoting the example from https://numpydoc.readthedocs.io/en/latest/example.html#example:
```
    Cite the relevant literature, e.g. [1]_.  You may also cite these
    references in the notes section above.

    .. [1] O. McNoleg, "The integration of GIS, remote sensing,
       expert systems and adaptive co-kriging for environmental habitat
       modelling of the Highland Haggis using object-oriented, fuzzy-logic
       and neural-network techniques," Computers & Geosciences, vol. 22,
       pp. 585-588, 1996.
```